src/v/rpc/types.h

```diff
@@ -182,6 +182,11 @@ enum class status : uint32_t {
     request_timeout = 408,
     server_error = 500,
     version_not_supported = 505,
+
+    // NOTE: older versions of Redpanda don't properly parse error codes they
```

Comment:
"older versions" will be hard to make sense of for future readers: do we mean "Redpanda <= 22.3.x"

Reply:
Done